### DISCUSSION TITLE
Templates shared with the workspace are not being displayed to users

### DISCUSSION THREAD

**Discussion by notfounnd** [outline/outline#7915] [Self Hosting]
I have in my Outline instance some document templates registered at the workspace level. I verified today that these doc templates that I have in the tool are not being displayed to be used by users.

Is this behavior normal?

To get around this I had to share the template to a view group of users.

**Comment by tommoor** [comment]
Doesn't sound normal, please file a bug with details. cc @apoorv-mishra

**Reply by notfounnd** [reply]
User view:

![image](https://github.com/user-attachments/assets/31c25549-7728-423b-9a33-2fcea9641d71)

Template config:

![image](https://github.com/user-attachments/assets/723d2d9c-b3fb-43ee-ab23-959d29fcc730)
